api/attestationconfigapi/fetcher.go

```diff
@@ -18,9 +18,23 @@ import (
 
 const cosignPublicKey = constants.CosignPublicKeyReleases
 
+var (
+	// AzureSEVSNP is the Azure SEV-SNP variant.
+	AzureSEVSNP Variant = variant.AzureSEVSNP{}
+	// AWSSEVSNP is the AWS SEV-SNP variant.
+	AWSSEVSNP Variant = variant.AWSSEVSNP{}
+	// GCPSEVSNP is the GCP SEV-SNP variant.
+	GCPSEVSNP Variant = variant.GCPSEVSNP{}
+)
+
+// Variant is a cloud provider specific attestation variant.
+type Variant interface {
+	String() string
+}
+
 // Fetcher fetches config API resources without authentication.
 type Fetcher interface {
-	FetchLatestVersion(ctx context.Context, attestation variant.Variant) (Entry, error)
+	FetchLatestVersion(ctx context.Context, attestation Variant) (Entry, error)
```

Comment:
continuum for example implements its own variant. I wanted to avoid also exposing this pkg.